packages/client/src/components/settings/FileTable.tsx

```diff
@@ -473,58 +472,58 @@ export const FileTable = (props: FileTableProps) => {
                             />
                         </Table.Cell>
                         <Table.Cell size="small">
-                            <TableSortLabel
-                                active={orderBy === headCell[0].id}
+                            <Table.Sort
+                                active={true} // sort icon is always visible
                                 direction={
                                     orderBy === headCell[0].id ? order : 'asc'
                                 }
                                 onClick={createSortHandler(headCell[0].id)}
                             >
+                                Name
                                 {orderBy === headCell[0].id ? (
-                                    <Box component="span" sx={visuallyHidden}>
+                                    <Box sx={visuallyHidden}>
                                         {order === 'desc'
                                             ? 'sorted descending'
                                             : 'sorted ascending'}
                                     </Box>
                                 ) : null}
-                            </TableSortLabel>
-                            {headCell[0].label}
+                            </Table.Sort>
                         </Table.Cell>
                         <Table.Cell size="small">
-                            <TableSortLabel
-                                active={orderBy === headCell[1].id}
+                            <Table.Sort
+                                active={true} // sort icon is always visible
                                 direction={
                                     orderBy === headCell[1].id ? order : 'asc'
                                 }
                                 onClick={createSortHandler(headCell[1].id)}
                             >
+                                Date Uploaded
                                 {orderBy === headCell[1].id ? (
-                                    <Box component="span" sx={visuallyHidden}>
+                                    <Box sx={visuallyHidden}>
```

Comment:
I understand the use of this, and think that it is fine. But I prefer not to add a package just for this accessibility issue, so could we get rid of the box and the import?